scripts/generate_road_colors.py

```diff
@@ -0,0 +1,211 @@
+from colormath.color_conversions import convert_color
+from colormath.color_objects import LabColor, LCHabColor, SpectralColor, sRGBColor, \
+XYZColor, LCHuvColor, IPTColor
+from colormath.color_diff import delta_e_cie2000
+import numpy
```

Comment:
In https://github.com/gravitystorm/openstreetmap-carto/blob/master/INSTALL.md ? Then I will also add note about Ruby that is necessary to run https://github.com/gravitystorm/openstreetmap-carto/blob/master/scripts/shop_values.rb .